recipes/mlpack/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set version = "3.0.2" %}
+
+
+package:
+    name: mlpack
+    version: {{ version }}
+
+source:
+    fn: mlpack-{{ version }}.tar.gz
+    url: http://www.mlpack.org/files/mlpack-{{ version }}.tar.gz
+    sha256: 69e81178839594049d84526b8f9612825f238f7cfe1f6d579bd6461ed48e5aa0
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - cython
+    - pip
+    - boost
+    - python
+    - numpy
+    - pandas  > 0.15.0
+    - armadillo
+    - pytest-runner
```

Comment:
Is this a build time dependency?

Reply:
Yes, I tried without it. However, their building process need the ``pytest-runner`` at build time